src/Tasks/Common/NETSdkMessage.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using System;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    /// <summary>
+    /// Provides a localizable mechanism for logging messages with different levels of importance from the SDK targets.
+    /// </summary>
+    public class NETSdkMessage : MessageBase
+    {
+        public string Importance { get; set; } = "Normal";
+
+        protected override void LogMessage(string message)
+        {
+            MessageImportance importance =
+                (MessageImportance)Enum.Parse(typeof(MessageImportance), Importance, true);
```

Comment:
Two nits if you keep this:

1. Use var, the type is obvious.
2. use named parameter for the boolean literal: `ignoreCase: true`